apps/extension/.env.testnet

```diff
@@ -1,3 +1,2 @@
 IDB_VERSION=42
-MINIFRONT_URL=https://app.testnet.penumbra.zone
```

Comment:
Really cool to finally get this .env file down to these two. Think we can rename to just `.env` and change the webpack config too:

```diff
- // Loads default vars from `.env.testnet` & `.env.mainnet` file in this directory.
- // Reference package.json build script.
- // TODO: add `.env.mainnet` when ready.
- //       const envPath = process.env['NODE_ENV'] === 'mainnet' ? '.env.mainnet' : '.env.testnet';
- //       dotenv.config({ path: envPath });
- dotenv.config({ path: '.env.testnet' });
+ // Loads default vars from `.env` file in this directory.
+ dotenv.config({ path: '.env' });

```